lib/sass.dart

```diff
@@ -15,12 +17,24 @@ import 'src/visitor/serialize.dart';
 /// If [color] is `true`, this will use terminal colors in warnings.
 ///
 /// Throws a [SassException] if conversion fails.
-String render(String path, {bool color: false}) {
+String render(String path,
+    {bool color: false, SyncPackageResolver packageResolver}) {
   var contents = readSassFile(path);
   var url = p.toUri(path);
   var sassTree = p.extension(path) == '.sass'
       ? new Stylesheet.parseSass(contents, url: url, color: color)
       : new Stylesheet.parseScss(contents, url: url, color: color);
-  var cssTree = evaluate(sassTree, color: color);
+  var cssTree =
+      evaluate(sassTree, color: color, packageResolver: packageResolver);
   return toCss(cssTree);
 }
+
+/// Loads the Sass file at [path], converts it to CSS, and returns the result in async way.
+/// It also uses the SyncPackageResolver to resolve package uri.
+///
+/// If [color] is `true`, this will use terminal colors in warnings.
+///
+/// Throws a [SassException] if conversion fails.
+Future<String> renderAsync(String path, {bool color: false}) async =>
```

Comment:
I don't think I want to add this. I don't think having a default for `packageResolver` is worth an extra top-level function.